azure/notebooks/kubernetes/README.md

```diff
@@ -0,0 +1,5 @@
+### Example Notebooks to run Rapids with Azure Kubernetes Service
+
+- Detailed instructions to set up Rapids with AKS is in the markdown file [Detailed_setup_guide.md](Detailed_setup_guide.md) . Go through this before you try to run any other notebooks.
+- Shorted example notebook using Dask + Rapids + XGBoost in [Dask_kube_XGBoost.ipynb](Dask_Kube_XGBoost.ipynb)
+- Full example with performance sweeps over multiple algorithms and bigger dataset in [Dask_Kube_cuML_Exploration_Full.ipynb](Dask_Kube_cuML_Exploration_Full.ipynb)
```

Comment:
```suggestion
- Full example with performance sweeps over multiple algorithms and larger dataset in [Dask_cuML_Exploration_Full.ipynb](Dask_Kube_cuML_Exploration_Full.ipynb)
```

Reply:
Let's also rename the other notebook

Reply:
Renamed to `MNMG_Exploration_Full.ipynb`